### DISCUSSION TITLE
Estimate total documents API

### DISCUSSION THREAD

**Discussion by ja7ad** [meilisearch/product#770] [Feedback & Feature Proposal]
In the documents API we don't have an API to estimate the total documents of an index.

We can just check total documents on the Get All documents  API.

https://www.meilisearch.com/docs/reference/api/documents#get-documents-with-post

Estimate total API support query param filter.

```shell
GET /indexes/{index_uid}/documents/estimate
```

**Queries:**

- filter as string

**Response:**

- total as integer

**Comment by ManyTheFish** [comment]
Hello @Ja7ad,

[The /stats route](https://www.meilisearch.com/docs/reference/api/stats#get-stats-of-all-indexes) may solve your issue,
is there anything that discourages the stats route usage?

Thanks!

**Reply by ja7ad** [reply]
> Hello @Ja7ad,
> 
> [The /stats route](https://www.meilisearch.com/docs/reference/api/stats#get-stats-of-all-indexes) may solve your issue, is there anything that discourages the stats route usage?
> 
> Thanks!

Thank you, Yes it solve my issue.
